raft/fsm.hh

```diff
@@ -355,6 +355,15 @@ public:
     bool is_candidate() const {
         return std::holds_alternative<candidate>(_state);
     }
+    std::string_view current_state() const {
+        static constexpr std::string_view leader_state = "Leader";
+        static constexpr std::string_view follower_state = "Follower";
+        static constexpr std::string_view candidate_state = "Candidate";
+        if(this->is_leader()){
```

Comment:
code style: spaces around `(this->is_leader())`.

Generally, please read https://github.com/scylladb/seastar/blob/master/coding-style.md and in particular look at the examples there and around the code.

Also `this->` is not necessary and we usually avoid it, unless it resolves some ambiguity (but in this case not really).